files/preStudies/frameworks.tex

```diff
@@ -0,0 +1,80 @@
+\section{Frameworks}
+
+This section will evaluate different frameworks for cross-platform development which could be used in this project. Some must have and nice to have capabilities will be
+	
+\textbf{Must have:}
+	\begin{itemize}
+	\item Cross platform capabilities
+	\item Wide community support
+	\item \textit{Bluetooth Low Energy} capabilities
+	\end{itemize}
+	
+\textbf{Nice to have:}
+	
+	\begin{itemize}
+	\item 1 platform to rule them all
+	\item type safety
+	\item reactive data
+	\item compiles to native code for increased performance
+	\end{itemize}
+	
+
+\subsubsection{Meteor}
+
+\textbf{language:} Javascript
+\\
+\textbf{Description:}
+\\
+\textit{Meteor} is a platform and framework for developing both web applications as well as mobile applications. It comes with \textit{Cordova} build in and is considered super fast for developing applications and prototyping. It comes with a lot of build-in tools and a package library for easy setup of things like account/user integration and login integrations of different kinds like Google and Facebook. It also comes with hot code push which is a tool for uploading new versions of your apps to the stores. \textit{Meteor} also have it's own CLI which is very useful when developing. \textit{Meteor} also allowes you to write your backend in plain javascript. Out of the box it integrates with mongo for data storage. It also uses DDP (Distributed Data Protocol) for sending data between the client and the server. This data is reactive and will be updated at each client who have subscribed to that data, if the data is changed on the server.
+\\
+Meteor comes with their own view layer named Blaze. Blaze can be substituted with React or AngularJS if either of them are preferred over Blaze.
```

Comment:
Meteor in italics.

"their" -> "its"
Again is the word Blaze used multiple times. Use placeholders instead, e.g. "... view layer named Blace. **This** can be..." and leave the last "over Blaze" part out.